ee/auth/prairietest.html.js

```diff
@@ -0,0 +1,66 @@
+const { html } = require('@prairielearn/html');
+const { renderEjs } = require('@prairielearn/html-ejs');
+
+const AuthPrairieTest = ({ jwt, prairieTestCallback, resLocals }) => {
+  return html`
+    <!DOCTYPE html>
+    <html lang="en">
+      <head>
+        ${renderEjs(__filename, "<%- include('../../pages/partials/head') %>", resLocals)}
+        <style>
+          .continue-card-container {
+            width: 100%;
+            max-width: 400px;
+          }
+        </style>
+      </head>
+      <body>
+        <form id="form" action="${prairieTestCallback}" method="POST">
+          <input type="hidden" name="jwt" value="${jwt}" />
+
+          <div class="continue-card-container mx-auto">
+            <div class="card continue-card m-3">
+              <div class="card-body d-flex flex-column align-items-center">
+                <div class="spinner-border mb-3" role="status">
+                  <span class="sr-only">Signing in...</span>
+                </div>
+                <h1 class="h4">PrairieTest authentication</h1>
+                <p>Signing in to PrairieTest...</p>
+                <button class="btn btn-success btn-block" type="submit" id="continue">
+                  Continue
+                </button>
+              </div>
+            </div>
+          </div>
+        </form>
+        <script>
+          // If JavaScript is enabled, immediately disable the button. But if
+          // it's disabled, the user can click the button to continue.
+          const continueButton = document.getElementById('continue');
+          continueButton.disabled = true;
+
+          (() => {
+            function submitForm() {
+              // Artificially slow down the submission so that the user can see
+              // a little more of what's happening.
+              setTimeout(() => {
+                const form = document.getElementById('form');
+                form.submit();
+              }, 1500);
```

Comment:
Do we really care to slow this down? I'm thinking about the login dance that AWS+Google do, where there appear to be about 20 redirects but at the end I just end up logged in, which is what I want, and I don't really care about what all the redirects are doing.

Reply:
Fair! I'll remove the delay.

Reply:
Done in 1b98b76.